clustered/integration-test/src/test/java/org/ehcache/clustered/management/DiagnosticTest.java

```diff
@@ -51,9 +51,9 @@ public void test_CACHE_MANAGER_CLOSED() throws Exception {
     int activePort = readTopology().serverStream().filter(Server::isActive).findFirst().get().getBindPort();
 
     Properties properties = new Properties();
-    properties.setProperty(ConnectionPropertyNames.CONNECTION_TIMEOUT, String.valueOf("5000"));
+    properties.setProperty(ConnectionPropertyNames.CONNECTION_TIMEOUT, String.valueOf("10000"));
     properties.setProperty(ConnectionPropertyNames.CONNECTION_NAME, "diagnostic");
-    properties.setProperty(PROP_REQUEST_TIMEOUT, "5000");
+    properties.setProperty(PROP_REQUEST_TIMEOUT, "10000");
```

Comment:
needed to to the same in tc-platform to make slow azure windows build pass. 10 sec was not enough in the case of tc-platform.

Reply:
I also needed to do that when using management clients directly in case you need: https://github.com/Terracotta-OSS/terracotta-platform/commit/990bd379672d07438f6465a7a95374d395144371#diff-2d8d456b4c6788aff9292dea30c0ff19R93-R99